src/node-file-trace.ts

```diff
@@ -137,15 +128,9 @@ export class Job {
       }, analysis === true ? {} : analysis);
     }
 
-    this.fileCache = cache && cache.fileCache || new Map();
-    this.statCache = cache && cache.statCache || new Map();
-    this.symlinkCache = cache && cache.symlinkCache || new Map();
     this.analysisCache = cache && cache.analysisCache || new Map();
```

Comment:
Should analysisCache also go in the `CachedFileSystem` class?

Reply:
It is not used there and right now it is only used in the job class.

If we move the cache we would need to move the analysis functionality there too, which I'm not sure fits into `CachedFileSystem `